scripts/updateSources.js

```diff
@@ -129,27 +119,6 @@ async function main() {
     content = content.replace(/\.attribute\("href", *"([^"]*)"\)/g, (...args) => {
       return `.attribute("href", "${wc}/${args[1]}")`;
     });
-    // Combo box - PreSelectedValueIT.selectedValueIsNotResetAfterClientResponse
-    // The test fails because when running in the project there is an iron-icon-set-svg element
```

Comment:
We no longer use `iron-iconset-svg` in V24 and the `IronIcon` was removed in #3934.
Furthermore, the actual ITs are already updated to use selectors specified below.